markdown/extensions/tables.py

```diff
@@ -160,6 +160,8 @@ class TableExtension(Extension):
 
     def extendMarkdown(self, md, md_globals):
         """ Add an instance of TableProcessor to BlockParser. """
+        if '|' not in md.ESCAPED_CHARS:
```

Comment:
This makes me wonder if `ESCAPED_CHARS` should be a set. Then it will always only contain one of each item and extensions don't need to check before appending. Just an idea.